comms/lighthouse/src/routes.ts

```diff
@@ -212,48 +79,6 @@ export function configureRoutes(app: express.Express, services: Services, option
         break
     }
   }
-
-  function mapLayerToJson(layer: Layer, globalMaxPerLayer: number | undefined, includeUserParcels: boolean = false) {
-    return {
-      name: layer.id,
-      usersCount: layer.peers.length,
-      maxUsers: layer.maxPeers ?? globalMaxPerLayer,
-      ...(includeUserParcels && {
-        usersParcels: layer.peers.map((it) => peersService.getPeerInfo(it).parcel).filter((it) => !!it)
-      })
-    }
-  }
-
-  function handleError(err: any, res, next) {
```

Comment:
I'm convinced this will be back eventually, when we expose islands through layers for inspection/debug purposes